crates/swc_ecma_loader/src/resolvers/node.rs

```diff
@@ -391,7 +409,11 @@ impl Resolve for NodeModulesResolver {
                         .and_then(|p| self.wrap(p))
                 } else {
                     self.resolve_node_modules(base_dir, target)
-                        .and_then(|p| self.wrap(p))
+                        .and_then(|path| {
+                            let current_directory = current_dir()?;
+                            let relative_path = diff_paths(path.unwrap(), current_directory);
```

Comment:
Can you remove this `unwrap()` too?

Reply:
yes but with `?` it does not work

Reply:
Yes b909630b183a5d7f9454dbb632faed79079b7f3f should fix the issue, but I need to add the context case for the error